internal/install/recipe_installer.go

```diff
@@ -88,6 +88,16 @@ func (i *RecipeInstaller) Install() error {
 		return i.fail(err)
 	}
 
+	infraAgentRecipe, err := i.fetchRecipeAndReportAvailable(m, infraAgentRecipeName)
```

Comment:
Is it relevant to fetch the infra and logging recipe when the `-c` flag is used?

Reply:
I think it is, since we want to show it as "Available" yet "Skipped" in the UI.

Reply:
Sorry to insist @ctrombley but I don't think it's valid to fetch the infraAgentRecipe and loggingRecipe at this level.
For one, those are NOT used when a recipe is provided explicitly with the -c option, which is NOT the workflow of what a user would do from the NR UI.
Second, if we look at the usages of those 2 recipe objects, they are only used later down in the ShouldInstallInfraAgent( ) and ShouldInstallLogging( ) method, and symmetrically those are not used for the ShouldInstallIntegrations( ) path which is the way the recipe path provided execute (-c option).
Consequently, could we just move the fetch of infra/logging closer to the scope where they are used in their respective ShouldInstallXxx( ) methods?